tutorials/Multi_objective_multi_fidelity_BO.ipynb

```diff
@@ -0,0 +1,1416 @@
+      "----------------------------\n",
+      "Iteration Number is  7\n",
+      "----------------------------\n",
+      "Iteration Number is  8\n",
+      "----------------------------\n",
+      "Iteration Number is  9\n",
+      "----------------------------\n",
+      "Iteration Number is  10\n",
+      "----------------------------\n",
+      "Iteration Number is  11\n",
+      "----------------------------\n",
+      "Iteration Number is  12\n",
+      "----------------------------\n",
+      "Iteration Number is  13\n",
+      "----------------------------\n",
+      "Iteration Number is  14\n",
+      "----------------------------\n",
+      "Iteration Number is  15\n",
+      "----------------------------\n",
+      "Iteration Number is  16\n",
+      "----------------------------\n",
+      "Iteration Number is  17\n",
+      "----------------------------\n",
+      "Iteration Number is  18\n",
+      "----------------------------\n",
+      "Iteration Number is  19\n",
+      "----------------------------\n",
+      "Iteration Number is  20\n",
+      "----------------------------\n",
+      "Iteration Number is  21\n",
+      "----------------------------\n",
+      "Iteration Number is  22\n",
+      "----------------------------\n",
+      "Iteration Number is  23\n",
+      "----------------------------\n",
+      "Iteration Number is  24\n",
+      "----------------------------\n",
+      "Iteration Number is  25\n",
+      "----------------------------\n",
+      "Iteration Number is  26\n",
+      "----------------------------\n",
+      "Iteration Number is  27\n",
+      "----------------------------\n",
+      "Iteration Number is  28\n",
+      "----------------------------\n",
+      "Iteration Number is  29\n",
+      "----------------------------\n",
+      "Iteration Number is  30\n"
+     ]
+    }
+   ],
+   "source": [
+    "# MOMF Opt run\n",
+    "from botorch import fit_gpytorch_model\n",
+    "from botorch.sampling.samplers import SobolQMCNormalSampler\n",
+    "\n",
+    "verbose = True\n",
+    "train_x = torch.zeros(n_INIT + n_BATCH * BATCH_SIZE, dim_x).to(**tkwargs)  # Intializing train_x to zero\n",
+    "train_obj = torch.zeros(n_INIT + n_BATCH * BATCH_SIZE, dim_y).to(**tkwargs)  # Intializing train_obj to zero\n",
+    "train_x[:n_INIT, :], train_obj[:n_INIT, :] = gen_init_data(dim_x, n_INIT, dim_y)  # Generate Initial Data\n",
+    "mll, model = initialize_model(train_x[:n_INIT, :], train_obj[:n_INIT, :])  # Initialize Model\n",
+    "for iteration in range(0, n_BATCH):\n",
+    "    # run N_BATCH rounds of BayesOpt after the initial random batch\n",
+    "    fit_gpytorch_model(mll)  # Fit the model\n",
+    "    momf_sampler = SobolQMCNormalSampler(num_samples=MC_SAMPLES)  # Generate Sampler\n",
+    "    # optimize acquisition functions and get new observations\n",
+    "    new_x, new_obj = optimize_MOMF_and_get_obs(\n",
+    "        model,\n",
+    "        train_obj[:, :],\n",
```

Comment:
Isn't this passing in a big tensor of zeros?

Reply:
You were right, I fixed this now.